lib/kitchen/directions/bake_chapter_content_module.rb

```diff
@@ -0,0 +1,13 @@
+module Kitchen
```

Comment:
are rubocop and lefthook running?  thought it would have made you put in module comments and a frozen string literal magic comment.